python/flask/secure-cookies/secure-cookies.yml

```diff
@@ -2,19 +2,73 @@
 rules:
   - id: secure-session-cookies
     patterns:
-      - pattern-not: |
-          $APP = flask.Flask(...)
-          ...
-          $APP.config.update(
-            ...,
-            SESSION_COOKIE_SECURE=True,
-            SESSION_COOKIE_HTTPONLY=True,
-            SESSION_COOKIE_SAMESITE="...",
-            ...
-          )
-      - pattern: |
-          $APP = flask.Flask(...)
-          ...
+      - pattern-either:
```

Comment:
Can the rule also detect the configuration update in one function call like in https://flask.palletsprojects.com/en/1.1.x/security/#security-cookie?

Reply:
Yes, check out this test: https://github.com/returntocorp/sgrep-rules/blob/flask/secure-cookies/python/flask/secure-cookies/session-secure-cookies-set.py